### ISSUE TITLE
legacy PB pages to be ported to PB-Jekyll

### ISSUE THREAD

**Issue by sdash-github** [PeanutBase/jekyll-peanutbase#26]
Many pages in legacy PB are linked from the tools panel of PB-Jekyll dev site. Some of these need to be ported to the jekyll site as  HTML pages.
(Links have to be individually checked and decided as to how it should be handled)

1. The Peanut Genome:  https://peanutbase.org/peanut_genome  
2. NIFA PCCGP Project: https://peanutbase.org/germplasm/nifa_pccgp  
3. Germplasm: https://dev.peanutbase.org/germplasm/  

These may wait until we decide:
- Marker-Assisted Selection: https://peanutbase.org/mas  
- Maps: https://peanutbase.org/chado/featuremap  (Tripal/Chado dependent)
- https://peanutbase.org/search/protein_domains (Tripal/Chado dependent)
- QTL Search: https://peanutbase.org/search/qtl  (Tripal/Chado dependent)

May not need be touched for now:
- Gigwa2.5 and  GCViT (not dependent on legacy site, so can stay as such for now)
- ArachisPheno(its own VM)

**Comment by svengato** [comment]
Some of these pages are now available for review. I will list each one in a separate message, along with notes.

**Comment by svengato** [comment]
The Peanut Genome
  https://dev.peanutbase.org/genome
Changes between genome assemblies v1 and v2
  https://dev.peanutbase.org/genome/peanut_genome_v1_v2

Issues:
1. Some links (IPGI, "Two diploid progenitors of cultivated peanut", GBrowse, JBrowse, data store assembly & annotation) still refer to the current peanutbase.org. Do we need to update these?
2. "IMPORTANT NOTE: the version 2.0 assembly (gnm2) for Arachis hypogaea var. Tifrunner has not been submitted to GenBank." Is this still true?
3. Links to Aiden Lab and DNA Zoo Announcement are missing/obsolete, let me know if alternatives exist.

Moved to issue #28.

**Comment by svengato** [comment]
NIFA PCCGP Project
  https://dev.peanutbase.org/germplasm/nifa_pccgp

Issue:
The Gigwa link is still under peanutbase.org. Are we planning to run Gigwa here? (The link would change to 'germplasm/gigwa' or something similar)

Moved to issue #29.

**Comment by svengato** [comment]
Marker-Assisted Selection
  https://dev.peanutbase.org/mas

Issues:
1. Trait pages: work in progress. I am thinking about whether these should go in a template, the trait managers could add information to a YAML file or something.
2. Still lots of peanutbase.org links.
3. Should the PDF files go in the repository? (Same question for dot plots in 'Changes between genome assemblies v1 and v2')

Moved to issue #27.

**Comment by sdash-github** [comment]
On gigwa let us wait until Thu PB meeting and will decide with Andrew's 
input, then we can change the link url. Somewhere in the issue or in a 
separate issue, you may list these as reminders lest they get invisible.

  * Gigwa2.5 and GCViT (not dependent on legacy site, so can stay as
    such for now)

**Comment by sdash-github** [comment]
May be we should create an issue for for each page I can comment on them 
as we go forward one by one.  Right now I don't have firm idea about MAS 
page and more traits with markers can be queried from  ArachisMine.

Sudhansu

**Comment by svengato** [comment]
Germplasm
https://dev.peanutbase.org/germplasm

Issues:
1. The Gigwa link is still under peanutbase.org (see above)
2. NIFA-PCCGP link is under Germplasm, is that correct?
3. What is aradu1_araip1_gnm1.div.HK28.sub10k.vcf? I see other diversity files in the data store, but not that one.
4. Interactive pedigree chart has gone missing.
5. Image Collection: work in progress.

Moved to issue #11.

**Comment by svengato** [comment]
Good idea, I will create separate issues. I have not yet looked at Maps, Protein Domains, QTL Search, and there may be others.

**Comment by svengato** [comment]
Another question is which links should appear in a new window/tab.

**Comment by svengato** [comment]
For convenience, I have temporarily added a red * by content that still needs updating, and a red # by links to pages created on the fly. These tags will disappear as each item gets fixed.

**Comment by svengato** [comment]
Committed first cuts at most of these on 9 May 2023. **Maps** and **QTL Search** are not yet ported.
